ai-baseline/app/services/initialization.py

```
@@ -0,0 +1,120 @@
+    path = f'/openai/deployments/{deployment_name}'
+    query_params = parse_qs(parsed.query)
+    query_params['api-version'] = [api_version]
+    return urlunparse((
+        parsed.scheme, parsed.netloc, path,
+        '', urlencode(query_params, doseq=True), ''
+    ))
+
+
+def _patch_llm_service_init():
+    """Patch LLMService.__init__ to use base URL for AzureOpenAI client."""
+    original_init = LLMService.__init__
```

Ruff (0.14.8)

Local variable `original_init` is assigned to but never used

Remove assignment to unused variable `original_init`

(F841)